models/stealcam/arbitrum/stealcam_arbitrum_fees.sql

```diff
@@ -0,0 +1,42 @@
+ {{
+  config(
+        alias='fees',
+        post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                    "project",
+                                    "stealcam",
+                                    \'["hildobby"]\') }}')
+}}
+
+SELECT blockchain,
+project,
+version,
+block_time,
+token_id,
+collection,
+CAST(platform_fee_amount_raw AS DOUBLE) AS platform_fee_amount_raw,
```

Comment:
can we push the cast statements to the base incremental model, rather than rolled up views?

Reply:
yeah, no problem, an old habit we're trying to clean up moving forward :)